src/graph/graph.js

```diff
@@ -174,6 +176,35 @@ class Graph {
     });
   }
 
+  /**
+   * Checks if the parsed input data is not empty
+   * @returns {void}
+   */
+  checkForData() {
```

Comment:
I would suggest to rename this to something like:
`hasData` or similar, and then when you call it:

```
if (!super.hasInputData()) { 
    super.renderNoData(); 
    return; 
}
```
In general, it is easier to have "positive" names without "not" and have the condition checking that "!hasData()"

Reply:
That's a very good point. I updated the commit. :)